### PR TITLE
Lazy load

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by jihdeh** [comment]
I have no way to test if the lazy load works, but until I merge it and test. Also before i merge this, i want to point out that there's a difference between npm install and yarn install. npm install creates a package-lock.json file, and yarn creates a yarn.lock file.

In this project, we're using yarn install. but I'll merge this now so i can move on with other testings.

File: client/debug.log [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[0531/090706.977:ERROR:crash_report_database_win.cc(427)] unexpected header
```
Comment (jihdeh) [inline client/debug.log]: delete this

Comment (luckeyodeh) [reply client/debug.log]: i have deleted the debug.log and pushed

File: package.json [hunk 2]
```diff
@@ -19,7 +19,8 @@
     "koa-mount": "git+https://github.com/jihdeh/mount#v3.0.3",
     "koa-response-time": "^2.0.0",
     "koa-router": "^7.4.0",
-    "koa-static": "^4.0.2"
+    "koa-static": "^4.0.2",
+    "react-lazyload": "^2.3.0"
```
Comment (jihdeh) [inline package.json]: You could have deleted this from here, as it's not used

**Pull request merged**